### ISSUE TITLE
make "name" column wider

### ISSUE THREAD

**Issue by 0xDones** [argoproj/argo-cd#21080]
# Summary

Whenever you click a group of Applications, you can see the following table. For longer names, if you have some prefixes within your organization, you can't see the full name, and you need to hover your mouse there.

![image](https://github.com/user-attachments/assets/92d47193-27ab-46ec-ae80-4f30d5994bf3)

# Motivation

Can easily view the Application name.

# Proposal

Could you make this column wider?

**Comment by 0xDones** [comment]
I think just making it wider is a quick win for now (maybe just one line of code change), I feel there's a lot of unused space there for the Status column, with no relevant information.

**Comment by KojoRising** [comment]
Fair enough. But the extra width may not be something everybody wants (it seems subjective, imho)

**Comment by 0xDones** [comment]
I don't think it's subjective since the application name is the most relevant piece of information on this entire table. We are not lacking space, half of the table is empty, we just need to use the space better there

**Comment by basava1139** [comment]
I agree with @0xDones—making the Name column wider is a practical solution for now. The application name is indeed the most important part of this table, and there’s sufficient unused space (like in the Status column) that can be better utilized.
